<launch>
  <node name="vosk_engine" pkg="speech_recognition_vosk" type="vosk_node.py" output="screen" />
  
  <arg name="lang" default="ja"/>
  <group if="$(eval lang=='ja')">
    <rosparam file="$(find speech_recognition_vosk)/cfg/params_ja.yaml" command="load" ns="" />
  </group>
  <group if="$(eval lang=='en')">
    <rosparam file="$(find speech_recognition_vosk)/cfg/params_en.yaml" command="load" ns="" />
  </group>
  <param name="recognition_mode" type="str" value="service"/>   <!-- please select "service" or "trigger", but please keep the "service" of this launch file -->
</launch>